container/build.py

```diff
@@ -13,11 +13,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""usage: build.py -d TYPE [-p PROJECT] [-c CONTAINER] [-t TAG]
+                   -v BAZEL_VERSION [-a] [-b BUCKET] [-h] [-m MAP] [-l]
 
-"""Builds a toolchain container, with Google Cloud Container Builder or locally.
+Builds the fully-loaded container, with Google Cloud Container Builder or
+locally.
+
+IF THIS SCRIPT IS CALLED FROM OUTSIDE OF THE BAZEL-TOOLCHAINS REPO, THE
```

Comment:
The issue I have with these docs is I would not know why I would call this script from a different repo. But this might be too much to include here. We might need a separate README to explain that better.

Reply:
Discussed offline. README should come in a future pr